Resources/Prototypes/Entities/Objects/Weapons/Melee/home_run_bat.yml

```
@@ -0,0 +1,34 @@
+- type: entity
+  parent: BaseBallBat
+  id: BaseBallBatHomeRun
+  name: home run bat
+  description: Heavy metal bat with an extra kick.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/home_run_bat.rsi
+    state: icon
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 11
+    bluntStaminaDamageFactor: 2.0
+    heavyRateModifier: 0.5
+    heavyDamageBaseModifier: 1.75
+    heavyStaminaCost: 25
+    maxTargets: 2
+    angle: 120
+    soundHit:
+      collection: ExplosionSmall
+  - type: MeleeRequiresWield # You can't hit a home run with one hand, jimbo.
+  - type: MeleeThrowOnHit
+    speed: 30
```

Oh, this datafield doesn't exist. 
```suggestion
    speed: 30
```

---

If you are talking about `speed`, the datafield is present in `MeleeThrowOnHitComponent`. https://github.com/Simple-Station/Einstein-Engines/blob/393116752ca73fba533ef404f0e7c0cafc1d6737/Content.Shared/Weapons/Melee/Components/MeleeThrowOnHitComponent.cs#L22

---

You're right, I clicked the wrong line.